### DISCUSSION TITLE
Why is sinytra a Fabric > NeoForge Project and not the other way around?

### DISCUSSION THREAD

**Discussion by PurpBatBoi** [Sinytra/Connector#1483] [Q&A]
Its good that this tool exists but why can't it also be used to have forge mods on the fabric? (constantly growing but overall still missing thousands of mods)

Should this compatibility layer be for both ways?

**Comment by WitherTM** [comment]
![image](https://github.com/user-attachments/assets/719bde9c-3c96-4a19-9507-609ddcdb2a22)

**Reply by PurpBatBoi** [reply]
the original point still valid, its weird that its just for using fabric on your forge installation and not the other way around

**Reply by coco875** [reply]
it's just fabric on forge are easier then forge on fabric and in 1.21 with neoforge more and more port decide to only support neoforge or make a port to neoforge like kubejs, ad astra, sodium, lithium, iris, etc
